server/scripts/standalone/publish-standalone.sh

```diff
@@ -10,8 +10,8 @@ build_artifact() {
     echo "[build]  Creating standalone extender server artifact at ${ARTIFACT_DIR}..."
     rm -rf ${ARTIFACT_DIR}
     mkdir -p ${ARTIFACT_DIR}
-    cp ${SERVER_DIR}/build/docker/extender-0.1.0.jar ${ARTIFACT_DIR}/extender.jar
```

Comment:
We should use the path for standalone builds. The jar files are copied in build.gradle in task `buildStandalone()`